### PR TITLE
esxi build

### PR DESCRIPTION
**Pull request by jjasghar**
- ESXi support

TODO:
- [x] README.md changes
- [ ] refactor some of redundant code

### CODE REVIEW

**Comment by edolnx** [comment]
Looks good to me, kinda exciting stuff.

File: recipes/esxi.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'pxe_dust::server'
+
+esxi = "/tmp/#{node['pxe_dust']['esxi_iso']}"
+
+directory '/mnt/loop' do
```
Comment (adamleff) [inline recipes/esxi.rb]: Maybe make this `/mnt/esxi-iso` or something more specific to the purpose so you don't worry about this conflicting with other mounts or user-initiated work?

File: templates/default/esxi-ks.cfg.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Accept the VMware End User License Agreement
+vmaccepteula
+# Configure a static IP Address on the first network adapter
+network --bootproto=dhcp --device=vmnic0
+# Set the root password for the DCUI and Tech Support Mode
+rootpw Ubuntu!!
```
Comment (adamleff) [inline templates/default/esxi-ks.cfg.erb]: Why not make this configurable? (and if you choose to do so, don't forget to update the README :smile:)

**Comment by adamleff** [comment]
Other than my nits, :+1:

**Pull request merged**